src/main/java/org/terasology/launcher/util/GuiUtils.java

```diff
@@ -102,10 +102,8 @@ public static File chooseDirectoryDialog(Stage owner, final File directory, fina
         }
 
         // directory proposal needs to be deleted if the user chose a different one
-        if (!directory.equals(selected) && !DirectoryUtils.containsFiles(directory)) {
-            if (!directory.delete()) {
-                logger.warn("Could not delete unused default directory! {}", directory);
-            }
+        if (!directory.equals(selected) && !DirectoryUtils.containsFiles(directory) && !directory.delete()) {
```

Comment:
I'd like to keep these for themselves. The warning should only be printed if there was an attempt to delete the file. The first if-expression determines whether the directory is in use, the second one checks the result of the delete operation.